app_dart/lib/src/service/github_service.dart

```diff
@@ -238,10 +238,12 @@ class GithubService {
 
   /// Gets the file content as UTF8 string of the file specified by the `path`
   /// in the repository.
-  Future<String> getFileContent(RepositorySlug slug, String path) async {
+  Future<String> getFileContent(RepositorySlug slug, String path, {String? ref}) async {
     ArgumentError.checkNotNull(slug);
     ArgumentError.checkNotNull(path);
-    final RepositoryContents contents = await github.repositories.getContents(slug, path);
+    final RepositoryContents contents = ref == null
```

Comment:
Why does this need a ternary? Ideally, this would look like

```dart
final RepositoryContents contents = await github.repositories.getContents(slug, path, ref: ref);
```

Reply:
Yeah this would involve many changes on flaky builders/issues handlers and their tests. I have updated those files and their tests to use the default branch as ref. Yeah previously i thought I could avoid these changes by supplying a tenary.